contentprovider.go

```diff
@@ -319,13 +320,28 @@ func matchScore(secs []DocumentSection, m *LineMatch) float64 {
 			}
 		}
 
+		if f.SymbolInfo != nil {
```

Comment:
In practice, is this field set? I remember us doing something around avoiding loading this unless it is a symbols query.

Reply:
With this change it is set whenever a match overlaps with a symbol. Previously only a symbol query would do the work to check the overlap between a match and a symbol. Now substr queries check too and enrich the matches that do overlap.

https://sourcegraph.com/github.com/sourcegraph/zoekt@acf74e5fb08ad45a4741c0eeb8e87f7053cd67c6/-/blob/contentprovider.go?L228%3A26=

Reply:
EDIT: With the last commits, this moved to `fillContentMatches`. Now all matches are enriched with symbol data if they match a symbol section.